bin/pt-stalk

```diff
@@ -820,27 +881,27 @@ collect() {
    collect_mysql_variables "$d/$p-variables" &
    sleep .5
 
-   local mysql_version="$(awk '/^version[^_]/{print substr($2,1,3)}' "$d/$p-variables")"
+   mysql_version="$(awk '/^version[^_]/{print substr($2,1,3)}' "$d/$p-variables")"
 
-   local mysql_error_log="$(awk '/^log_error/{print $2}' "$d/$p-variables")"
+   mysql_error_log="$(awk '/^log_error\s/{print $2}' "$d/$p-variables")"
    if [ -z "$mysql_error_log" -a "$mysqld_pid" ]; then
+      log $mysqld_pid
       mysql_error_log="$(ls -l /proc/$mysqld_pid/fd | awk '/ 2 ->/{print $NF}')"
    fi
 
-   local tail_error_log_pid=""
    if [ "$mysql_error_log" -a ! "$OPT_MYSQL_ONLY" ]; then
       log "The MySQL error log seems to be $mysql_error_log"
       tail -f "$mysql_error_log" >"$d/$p-log_error" &
       tail_error_log_pid=$!
 
-      $CMD_MYSQLADMIN $EXT_ARGV debug
```

Comment:
why we delete this?

Reply:
This is the fix for https://jira.percona.com/browse/PT-1340 that was removed by the fix for PT-1398